include/wxcharts/wxareachart.h

```diff
@@ -36,14 +36,16 @@
 #ifndef _WX_CHARTS_WXAREACHART_H_
```

Comment:
Why are all the header files being moved to a new directory. Other wxWidgets libraries follow this convention so I don't see a reason to change it.

Reply:
It based on a earlier version of wxCharts, I'll fix it.